services/src/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/FlowCrudSpec.groovy

```diff
@@ -713,6 +718,62 @@ class FlowCrudSpec extends BaseSpecification {
         "an unmetered"  | 0
     }
 
+    @Unroll
+    @Tags(VIRTUAL)
+    def "Able to create an one-switch flow on a DEACTIVATED switch"(FlowPayload flow) {
+        given: "A deactivated switch"
+        def sw = topology.getActiveSwitches().find { it.dpId == flow.source.switchDpId }
+        lockKeeper.knockoutSwitch(sw)
+        TimeUnit.SECONDS.sleep(10) // for no good reason it fails without sleep
```

Comment:
We have tons of tests where this operation works fine. Not sure why it should require additional sleep here